js/languages/en-US.json

```diff
@@ -546,7 +546,8 @@
     "goodSMTPAuthentication": "You have successfully connected to the SMTP server.",
     "tagIsTooLongHeadline": "The tag is too long",
     "tagIsTooLongBody": "Tags cannot be more than %{smart_count} character long. |||| Tags cannot be more than %{smart_count} characters long.",
-    "retryingConnection": "The connection to your node was lost. Retrying connection attempt %{attempt} of %{total}."
+    "retryingConnection": "The connection to your node was lost. Retrying connection attempt %{attempt} of %{total}.",
+    "peersFail": "Call to peers API failed."
```

Comment:
"Call to the peers API failed."

I know that's nitpicky, but it confuses our translators if we don't use correct grammar (not that we're guilt-free on that front at all).